iroh-sync/src/actor.rs

```diff
@@ -775,24 +823,33 @@ impl Actor {
 struct OpenReplicas(HashMap<NamespaceId, OpenReplica>);
 
 impl OpenReplicas {
-    fn replica(&mut self, namespace: &NamespaceId) -> Result<&mut Replica<StoreInstance>> {
-        self.get_mut(namespace).map(|state| &mut state.replica)
-    }
-
-    fn get_mut(&mut self, namespace: &NamespaceId) -> Result<&mut OpenReplica> {
-        self.0.get_mut(namespace).context("replica not open")
+    fn replica<'a, 'b>(
+        &'a mut self,
+        namespace: NamespaceId,
+        store: &'b mut Store,
+    ) -> Result<Replica<StoreInstance<'b>, &'a mut ReplicaInfo>> {
```

Comment:
Maybe add a type alias for `Replica<StoreInstance<'b>, &'a mut ReplicaInfo>`, this is used a lot